src/App.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
I think we are missing Jest in our setup. I know that we will be missing some tests but at least we can do some.